.github/ISSUE_TEMPLATE/icon_removal.yml

```diff
@@ -0,0 +1,30 @@
+name: Icon removal
+description: Report an icon for removal
+title: "Remove: "
+labels: [removal request]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        Before opening a new issue, make sure it isn't covered by an existing issue.
+        Please search for [issues with the `removal request` label][removal-request-issues]
```

Comment:
I would expect the `removal request` label to be removed once the removal has been confirmed. From that perspective this search should probably include the `breaking change` label. What do you think about that @sachinraja?

Reply:
I agree

Reply:
Apologies for the confusion, I wasn't suggesting to [add the breaking change label](https://github.com/simple-icons/simple-icons/pull/6552/commits/360ef0b90f40d5b8f9291a898bc68c91b2b42438) to this template, but rather that this link should include `breaking change` labels in the search result.

I personally wouldn't even add `breaking change` by default. I'd add it if the removal request has been verified, rather than having to remove it when the removal request turns out to be invalid. But it can go either way so whatever you think makes the most sense works for me.

Reply:
That makes more sense to me. It's more clear to people that it's just a request and no changes are being made yet.